screens/Explore/ExploreScreen.tsx

```diff
@@ -12,7 +12,32 @@ import {
   useSubscribedCourses,
 } from "@/hooks/query";
 import LoadingScreen from "@/components/Loading/LoadingScreen";
-import { Course } from "@/types";
+import { Course } from "@/types/index";
+import { LinearGradient } from "expo-linear-gradient";
+import { colors } from "@/theme/colors";
+import MaterialCommunityIcons from "@expo/vector-icons/MaterialCommunityIcons";
+
+const RecommendationBadge = ({ children }: PropsWithChildren) => {
+  return (
+    <View className="relative my-3">
+      <View className="absolute -top-3 right-3 z-10 overflow-hidden rounded-xl shadow-lg">
```

Comment:
@MadsHeilmann Are you sure it should be absolute position? In 99% of cases that results in strange UIs.